### PR TITLE
chore: upgrade deps, svelte 4 and others

### LINKED ISSUE #239: ✨ Update repo and base templates to svelte 4
### Describe the problem

svelte 4 is out and we should update!

### Describe the solution you would like

Update everything

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by paoloricciuti**
Just a dep routine update ...it also closes #239

### CODE REVIEW

**Comment by paoloricciuti** [comment]
Everything seems to be working fine @SarcevicAntonio ...whenever you have time just do a quick tour to verify that everything works find

File: src/lib/components/Errors.svelte [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 <ul>
 	{#each [...diagnostics] as diagnostic}
-		<li class={diagnostic.type} transition:slide={{ delay: 250, duration: 250 }}>
+		<li class={diagnostic.type} transition:slide|global={{ delay: 250, duration: 250 }}>
```
Comment (SarcevicAntonio) [inline src/lib/components/Errors.svelte:16]: Not sure we need this here but could also not matter

**Review by SarcevicAntonio — APPROVED** [review]
Looks good

File: src/lib/components/Errors.svelte (hunk 1 above)
Comment (paoloricciuti) [reply src/lib/components/Errors.svelte:16]: It was added by the svelte migration tool...and I guess, why not 🤷‍♂️

**Pull request merged**